hicommonwealth/commonwealth
# Deleted GatingGrowl from Sublayout

## Link to Issue
Closes: #6516 

## Description of Changes
- Delete `GatingGrowl` component from `Sublayout` to ensure it no longer shows

## "How We Fixed It"

- Removed the import and usage of `<GatingGrowl />`

## Test Plan
- Ensure your localStorage `dontShowGatingGrowlEver` is set to false or deleted
- Navigate to a community
- Ensure the gating growl does not show
